core/src/main/java/org/infinispan/notifications/cachelistener/event/Event.java

```diff
@@ -9,7 +9,7 @@
  * @since 4.0
  */
 public interface Event<K, V> {
-   static enum Type {
+   enum Type {
```

Comment:
I am guessing it is possibly because of not injecting values in a wrapped filter maybe?